whitepaper/whitepaper.md

```diff
@@ -1553,7 +1595,10 @@ Web 3.0 applications (decentralized apps or *DApps*) which interact with blockch
 
 **Tx submitter**: The entity submits the remote transaction to the API server plug-in on one of the ledgers.
 
-**API Server**: A module of Hyperledger Cactus which provides a unified interface to control/monitor Blockchain ledger behind it.
+**API Server**: A Cactus package that is the backbone of the plugin architecture and the host component for all plugins with the ability to automaticaly wire up plugins that come with their own web services (REST or asynchronous APIs through HTTP or WebSockets for example)
+
+**Cactus Node**: A set of identically configured API servers behind a single network host where the set size can range from 1 to infinity with the practical maximum being much lower of course. This logical distinction between node and API server is important because it allows consortium members to abstract away their private infrastructure details from the public consortium definition. For example if a consortium member wants to have a highly available, high throughput service, they will be forced to run a cluster of API servers behind a load balancer and/or reverse proxy to achieve these system properties and their API servers may also be in an auto-scaling group of a cloud provider or (in the future) even run as Lambda functions. To avoid having to update the consortium definiton (which requires a potentially costly consesus from other members) every time let's say an auto-scaling group adds a new API server to a node, the consortium member can define their presence in the consortium by declaring a single `Cactus Node` and then customize the underlying deployment as they see fit so long as they ensure that the previously agreed upon keys are used by the node and it is indeed accessible through the network host as declared by the `Cactus Node`.
```

Comment:
Instead of "set size can range...." we could say "where the set size is customizable". I would make a paragraph from "For example" to separate the example from the definition.
Apart from that looks very clean and concise!

Reply:
@RafaelAPB Makes sense, cheers! See https://github.com/hyperledger/cactus/pull/331/commits/a0ac20f755b4958be59cdb92da163f54bb516bbc